epcis_event_hash_generator/hash_generator.py

```diff
@@ -147,6 +149,28 @@ def compute_prehash_from_file(path, enforce=None):
     else:
         logging.error("Filename '%s' ending not recognized.", path)
 
+    return compute_prehash_from_events(events)
+
+
+def compute_prehash_from_json_str(json, enforce=None):
+    """Read EPCIS document and generate pre-hashe strings.
+    Use enforce = "XML" or "JSON" to ignore file ending.
+    """
+
+    events = read_jsonStr(json)
+
+    return compute_prehash_from_events(events)
+
+
+def compute_prehash_from_xml_str(xmlStr, enforce=None):
```

Comment:
enforce is not used

Reply:
as method is specific for xml hence enforce variable is not required. Have removed it.